docs/metrics.asciidoc

```diff
@@ -81,7 +81,7 @@ The total virtual memory the process has.
 [[metrics-golang]]
 === Go runtime metrics
 
-The Go agent reports various Go runtime metrics.
+The Go {agent} reports various Go runtime metrics.
```

Comment:
I would double check to ensure that the correct Terminology is Go Elastic Agent. If so, you can leave as.

Reply:
```suggestion
The Go agent reports various Go runtime metrics.
```